### PR TITLE
Doc updates for 1.0 release

### PR DESCRIPTION
**Pull request by markxnelson**

### CODE REVIEW

**Review by andytael — APPROVED** [review]

File: docs-source/mbaas/content/on-premises/ol8/_index.md [hunk 1]
```diff
@@ -12,20 +12,30 @@ Read [On-Premises](../index.md) and ensure that your desktop meets the minimum s
...
 Set the default Python3 to Python 3.9:
 
 ```bash
-sudo alternatives --set python3 /usr/bin/python3.9
+alternatives --set python3 /usr/bin/python3.9
+```
+
+### Install MiniKube
+
+As the `root` user, install minikube:
```
Comment (rhondaday) [inline docs-source/mbaas/content/on-premises/ol8/_index.md:34]: Please change "... install minikube:" to "... install Minikube:"

File: docs-source/mbaas/content/on-premises/ol8/_index.md [hunk 2]
```diff
@@ -48,23 +58,16 @@ As the `obaas` user, take these steps:
...
 
-### Install and Start Minikube
-
-As the `root` user, process these commands:
-
-```bash
-curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
-install minikube-linux-amd64 /usr/local/bin/minikube
-```
+### Start Minikube
```
Comment (rhondaday) [inline docs-source/mbaas/content/on-premises/ol8/_index.md:70]: Change "Minikube" to "MiniKube"

File: docs-source/mbaas/content/on-premises/ol8/_index.md [hunk 3]
```diff
@@ -12,20 +12,30 @@ Read [On-Premises](../index.md) and ensure that your desktop meets the minimum s
...
 ```
 
 Set the default Python3 to Python 3.9:
 
 ```bash
-sudo alternatives --set python3 /usr/bin/python3.9
+alternatives --set python3 /usr/bin/python3.9
+```
+
+### Install MiniKube
```
Comment (rhondaday) [inline docs-source/mbaas/content/on-premises/ol8/_index.md:32]: "MiniKube" should be "Minikube"

File: docs-source/spring/content/on-premises/ol8/_index.md [hunk 4]
```diff
@@ -28,6 +27,15 @@ Set the default Python3 to Python 3.9 by running this command:
 sudo alternatives --set python3 /usr/bin/python3.9
 ```
 
+### Install MiniKube
```
Comment (rhondaday) [inline docs-source/spring/content/on-premises/ol8/_index.md:30]: "MiniKube" should be "Minikube"

File: docs-source/spring/content/on-premises/ol8/_index.md [hunk 5]
```diff
@@ -28,6 +27,15 @@ Set the default Python3 to Python 3.9 by running this command:
...
+
+As the `root` user, install minikube:
```
Comment (rhondaday) [inline docs-source/spring/content/on-premises/ol8/_index.md]: "minikube" should be "Minikube"

File: docs-source/spring/content/_index.md [hunk 6]
```diff
@@ -14,37 +14,58 @@ In addition to an Oracle Autonomous Database Serverless instance, the following
...
 - OpenTelemetry Collector (including automatic instrumentation).
 
+## Learn more, try it out with CloudBank!
+
+To learn more about deploying and using Oracle Backend for Spring Boot and Microservices, we recommend our
+[CloudBank](https://bit.ly/CloudBankOnOBaaS) self-paced, on-demand hands-on lab.
+
+![](./cloudbank-hol.png)
+
+In the [CloudBank](https://bit.ly/CloudBankOnOBaaS) hands-on lab you can learn how to:
```
Comment (rhondaday) [inline docs-source/spring/content/_index.md:52]: Add a comma after "lab".

File: docs-source/spring/content/_index.md [hunk 7]
```diff
@@ -14,37 +14,58 @@ In addition to an Oracle Autonomous Database Serverless instance, the following
...
+
+- Install Oracle Backend for Spring Boot and Microservices.
+- Set up a development environment for Spring Boot.
+- Build Spring Boot microservices from scratch using Spring Web to create REST services.
```
Comment (rhondaday) [inline docs-source/spring/content/_index.md:56]: Change "REST" to "Representational State Transfer (REST)"

File: docs-source/spring/content/_index.md [hunk 8]
```diff
@@ -14,37 +14,58 @@ In addition to an Oracle Autonomous Database Serverless instance, the following
...
+- Use service discovery and client-side load balancing.
+- Use Spring Actuator to allow monitoring of services.
+- Create services that use asynchronous messaging (with JMS) instead of REST.
```
Comment (rhondaday) [inline docs-source/spring/content/_index.md:59]: Change "JMS" to "Java Message Service (JMS)"

File: docs-source/spring/content/database/_index.md [hunk 9]
```diff
@@ -15,12 +15,11 @@ resources:
...
 ---
 
-The Oracle Backend for Spring Boot includes an Oracle database. An instance of Oracle Autonomous Database Serverless is created during
-installation.
+The Oracle Backend for Spring Boot and Microservices includes an Oracle database. An instance of Oracle Autonomous Database Serverless is created during installation.
 
-If you chose the **Secure Access from Anywhere** option for your database during installation (or just accepted the default), then you can use
-the **Database Actions** web user interface to work with your database. If you chose the **Private** option, you need to use Bastion to access
-the database.
+> **_NOTE:_** Oracle recommends that you install your own database(s) for your production applications, the database provisioned is used for Oracle Backend for Spring Boot metadata and can be used for developement.
```
Comment (rhondaday) [inline docs-source/spring/content/database/_index.md:20]: Change "database(s)" to "databases", change the comma to a period, and init cap "The".

File: docs-source/spring/content/database/_index.md [hunk 10]
```diff
@@ -59,46 +55,44 @@ Process the following steps:
 
 1. Create a Dynamic Port Forwarding (SOCKS5) Session using the Bastion service.
 
-    Start with Autonomous Database (ADB) access that was created with private endpoint access only in accordance with security guide lines.
-	For ADB to run SQL commands, you need to establish a session between your local machine and ADB using the Bastion service.
+    Start with Autonomous Database (ADB) access that was created with private endpoint access only in accordance with security guide lines.  For ADB to run SQL commands, you need to establish a session between your local machine and ADB using the Bastion service.
```
Comment (rhondaday) [inline docs-source/spring/content/database/_index.md:58]: Change "guide lines" to "guidelines".

File: docs-source/spring/content/development/cli/_index.md [hunk 11]
```diff
@@ -2,14 +2,14 @@
 title: "OBaaS CLI"
 ---
 
-The Oracle Backend for Spring Boot offers a command-line interface (CLI), `oractl`. The CLI commands simplify the deployment of
+The Oracle Backend for Spring Boot and Microservices offers a command-line interface (CLI), `oractl`. The CLI commands simplify the deployment of
```
Comment (rhondaday) [inline docs-source/spring/content/development/cli/_index.md:5]: I think "Microservices" in the second sentence should be "microservices".

File: docs-source/spring/content/on-premises/_index.md [hunk 12]
```diff
@@ -50,70 +48,92 @@ appropriate documentation for examples of installing and defining the _Desktop_
...
 ...
 ```
-Specify the URL or authentication credentials for your Container Repository in `pull_registry_url`, `push_registry_url`, `registry_username`
-and `registry_password`.  
 
-For the `registry_auth` section, manually log in to your repository and copy the values found in the previously created file, which is often
-found in `$HOME/.config/containers/auth.json`
+Specify the URL or authentication credentials for your Container Repository in `registry_pull_url`, `registry_push_url`, `registry_username`, and `registry_password`.
+
+For the `registry_pull_auth` and `registry_push_auth` sections, manually log into your repository and copy the values found in created file, located in `$HOME/.config/containers/auth.json`
```
Comment (rhondaday) [inline docs-source/spring/content/on-premises/_index.md:136]: "... found in the created file, ..."

File: docs-source/spring/content/on-premises/_index.md [hunk 13]
```diff
@@ -135,7 +155,7 @@ source ./activate.env
 
 ### Desktop Playbook
 
-If this is an _Estate_ installation, the infrastructure should be manually defined as previously stated.  
+If this is an _Estate_ installation, the infrastructure should be manually defined as previously stated.
```
Comment (rhondaday) [inline docs-source/spring/content/on-premises/_index.md:158]: "... then the infrastructure ..."

File: docs-source/spring/content/on-premises/_index.md [hunk 14]
```diff
@@ -135,7 +155,7 @@ source ./activate.env
...
 
 If this is a _Desktop_ installation, run the Helper Playbook to define the infrastructure. For example:
```
Comment (rhondaday) [inline docs-source/spring/content/on-premises/_index.md:160]: "... then run the Helper ..."

File: docs-source/spring/content/platform/vscode-plugin/_index.md [hunk 15]
```diff
@@ -67,18 +67,18 @@ To restart the plugin and proceed with setting the Kubernetes configuration file
 
 ### How to Access the Cluster
 
-Until you create a dedicated `ssh` tunnel to the Kubernetes cluster, and if you do not connect to Oracle Backend for Spring Boot adminstrative
-services, you cannot browse resources included in the Oracle Backend for Spring Boot deployment. To do this, follow these steps:
+Until you create a dedicated `ssh` tunnel to the Kubernetes cluster, and if you do not connect to Oracle Backend for Spring Boot and Microservices adminstrative
```
Comment (rhondaday) [inline docs-source/spring/content/platform/vscode-plugin/_index.md:71]: typo: adminstrative

File: docs-source/spring/content/release-notes/_index.md [hunk 16]
```diff
@@ -2,6 +2,53 @@
...
 
+## Version 1.0.0, October 15, 2023
+
+This is the first production release.
+
+Notes:
+
+* All Spring platform components upgraded to Spring Boot 3.x, Spring 6.x and Spring Cloud 2022.0.4 versions.
+* Spring Native (with GraalVM native image ahead of time compilation) is the default/recommended
+  deployment option for microservices.  JVM continues to be available as an option.
```
Comment (rhondaday) [inline docs-source/spring/content/release-notes/_index.md:13]: Delete the extra spaces between sentences.

File: docs-source/spring/content/release-notes/_index.md [hunk 17]
```diff
@@ -2,6 +2,53 @@
...
+* Loki and Promtail added for logging, Spring Boot dashboard updated to show metrics and logs on same dashboard.
```
Comment (rhondaday) [inline docs-source/spring/content/release-notes/_index.md:14]: " ... on the same dashboard."

File: docs-source/spring/content/security/_index.md [hunk 18]
```diff
@@ -2,32 +2,22 @@
...
 
-The Oracle Backend for Spring Boot has the following security characteristics:
+The Oracle Backend for Spring Boot and Microservices has the following security characteristics:
 
-- All access to the database is done using the Mutual Transport Layer Security (mTLS) database wallet. The user name, password, and URL are
-  stored in Kubernetes secrets.
+- All access to the database is done using the Mutual Transport Layer Security (mTLS) database wallet. The user name, password, and URL are stored in Kubernetes secrets.
 
-- Accessing the dashboards can only be done using port-forwarding.
+- Accessing the dashboards can only be done using port-forwarding. The passwords for the dashboards are provied by the installer or autogenerated and stored in k8s secrets.
```
Comment (rhondaday) [inline docs-source/spring/content/security/_index.md:9]: typo: provied
"k8s" should be "K8s"

File: docs-source/spring/content/security/_index.md [hunk 19]
```diff
@@ -2,32 +2,22 @@
...
+- A customer application can only access services running in the platform using the Apache APISIX Gateway. [APISIX Gateway Documentation](../platform/apigw)
 
-    ```bash
-    kubectl -n ingress-nginx get svc ingress-nginx-controller
-    ```
+- The customer application can be secured using Spring Security (for example, Spring OAuth 2.0 Resource Server). An external authorization server is recommended (for example Okta or Auth0).
 
-- The customer application can be secured using Spring Security (for example, Spring OAuth 2.0 Resource Server). An external authorization
-  server is recommended (for example Okta or Auth0).
+- Oracle Backend for Spring Boot and Microservices provides a AuthZ server which is an engine to authenticate and authorize requests to various components in Oracle Backend for Spring Boot. The end user can manages users using REST Endpoints. [AuthZ Server Documentation](../security/azn-server)
```
Comment (rhondaday) [inline docs-source/spring/content/security/_index.md:21]: First sentence: "... provides an AuthZ ... "
First sentence: Should "and Microservices" be added after "Spring Boot"?
Second sentence: "The end user can manage users ..."

File: docs-source/spring/content/security/azn-server/_index.md [hunk 20]
```diff
@@ -0,0 +1,136 @@
...
+    title: "Authorization Server Architecture"
+  - name: azn-stack-app-info
+    src: "azn-stack-app-info.png"
+    title: "Application Information"
+  - name: azn-passwords-console
+    src: "azn-passwords-console.png"
+    title: "AZN User passwords"
+---
+
+The Authorization Server is an engine to authenticate and authorize requests to various components in Oracle Backend for Spring Boot and Microservices. The end user can manages users using REST Endpoints.
```
Comment (rhondaday) [inline docs-source/spring/content/security/azn-server/_index.md:15]: Second sentence: "The end user can manage users ..."

File: docs-source/spring/content/security/azn-server/_index.md [hunk 21]
```diff
@@ -0,0 +1,136 @@
...
+
+> **_NOTE:_** Oracle recommends that you change the default passwords for the default created users.
+
+## Users & Roles
+
+When deploying Oracle Backend for Spring Boot two users are created with the following roles:
```
Comment (rhondaday) [inline docs-source/spring/content/security/azn-server/_index.md:21]: Add a comma after "Boot".

File: docs-source/spring/content/security/azn-server/_index.md [hunk 22]
```diff
@@ -0,0 +1,136 @@
...
+
+| User Name     | Assigned Roles        |
+|---------------|-----------------------|
+| obaas-admin   | ROLE_ADMIN, ROLE_USER |
+| obaas-user    | ROLE_USER             |
+
+All the users are stored in the database deployed when installing Oracle Backend for Spring Boot and Microservices. The Roles determines what the user is allowed to do in the environment. The allowed roles are `ROLE_ADMIN` and `ROLE_USER`.
```
Comment (rhondaday) [inline docs-source/spring/content/security/azn-server/_index.md:28]: First sentence: "All users stored in the database are deployed ..."
Second sentence: "The roles determine what ..."

File: docs-source/spring/content/security/azn-server/_index.md [hunk 23]
```diff
@@ -0,0 +1,136 @@
...
+
+> **_NOTE:_** See each components documentation about the roles and authorities.
+
+The assigned passwords (either auto generated or provided by the installer) can be viewed in the OCI Console (ORM homepage). Click on **Application Information**.
+
+<!-- spellchecker-disable -->
+{{< img name="azn-stack-app-info" >}}
+<!-- spellchecker-enable -->
+
+If you click on **Unlock** the password for the obaas-admin and obaas-user can be displayed.
```
Comment (rhondaday) [inline docs-source/spring/content/security/azn-server/_index.md:38]: Add a comma after "Unlock".

File: docs-source/spring/content/security/azn-server/_index.md [hunk 24]
```diff
@@ -0,0 +1,136 @@
...
+kubectl get secret -n azn-server oractl-passwords -o jsonpath='{.data.admin}' | base64 -d
+```
+
+For `obaas-user`:
+
+```shell
+kubectl get secret -n azn-server oractl-passwords -o jsonpath='{.data.user}' | base64 -d
+```
+
+## User Management REST endpoints
```
Comment (rhondaday) [inline docs-source/spring/content/security/azn-server/_index.md:58]: Init cap "Endpoints"

File: docs-source/spring/content/security/azn-server/_index.md [hunk 25]
```diff
@@ -0,0 +1,136 @@
...
+
+The following REST endpoints are available to manage users. The table lists which minimum required Role is needed to perform the operation.
```
Comment (rhondaday) [inline docs-source/spring/content/security/azn-server/_index.md:60]: Second sentence: "The table lists the minimum required role that is ..."

File: docs-source/spring/content/security/azn-server/_index.md [hunk 26]
```diff
@@ -0,0 +1,136 @@
...
+
+| End point                                         | Method | Description                                     | Minimum required Role |
```
Comment (rhondaday) [inline docs-source/spring/content/security/azn-server/_index.md:62]: "End point" should be "Endpoint"
"required" should be "Required"

File: docs-source/spring/content/security/azn-server/_index.md [hunk 27]
```diff
@@ -0,0 +1,136 @@
...
+| /user/api/v1/findUser                             | GET    | Find all users                                  | ROLE_ADMIN            |
+| /user/api/v1/findUser?username=\<username\>       | GET    | Find a user with the username \<username\>      | ROLE_ADMIN            |
+| /user/api/v1/createUser                           | POST   | Create a user                                   | ROLE_ADMIN            |
+| /user/api/v1/updatePassword                       | PUT    | Update a password for a user. A user with<br>Role ROLE_ADMIN can update any users password | ROLE_USER |
+| /user/api/v1/deleteUsername?username=\<username\> | DELETE | Delete a user with username \<username\>        | ROLE_ADMIN            |
+| /user/api/v1/deleteId?id=\<id\>                   | DELETE | Delete a user with the id \<id\>                | ROLE_ADMIN            |
+
+### User Management REST Endpoints
+
+In all examples below you need to replace `<username>:<password>` with your username and password. The examples are using `curl` to interact with the REST endpoints. They also requires that you have opened a tunnel on port 8080 to either the `azn-server` or `obaas-admin` service. For example
```
Comment (rhondaday) [inline docs-source/spring/content/security/azn-server/_index.md:73]: First sentence: "In all of the following examples, you ..."
Third sentence: "They also require that you ..."
Add a : after "example"

File: docs-source/spring/content/security/azn-server/_index.md [hunk 28]
```diff
@@ -0,0 +1,136 @@
...
+
+#### /user/api/v1/findUser?username=\<username\>
+
+```shell
+curl -i -u <username>:<password> 'http://localhost:8080/user/api/v1/findUser?username=obaas-admin'
+```
+
+#### /user/api/v1/createUser
+
+When creating a user the following Roles are allowed: `ROLE_ADMIN` and `ROLE_USER`.
```
Comment (rhondaday) [inline docs-source/spring/content/security/azn-server/_index.md:93]: Add a comma after "user" and change "Roles" to "roles"

File: docs-source/spring/content/security/azn-server/_index.md [hunk 29]
```diff
@@ -0,0 +1,136 @@
...
+    http://localhost:8080/user/api/v1/deleteId?id=<userid_to_be_deleted>
+```
+
+## Architecture
+
+The following picture shows how the Authentication Server is used for AuthZ for the following modules:
+
+- OBaaS Admin (OBaaS CLI server module)
+- Config Server (Manages Config Server Entries)
+- AZN Server (AUthentication Server User Management)
```
Comment (rhondaday) [inline docs-source/spring/content/security/azn-server/_index.md:131]: typo: Authentication (not AUthentication)

File: docs-source/spring/content/security/azn-server/_index.md [hunk 30]
```diff
@@ -0,0 +1,136 @@
...
+curl -u obaas-admin:password -i -X DELETE \
+    http://localhost:8080/user/api/v1/deleteId?id=<userid_to_be_deleted>
+```
+
+## Architecture
+
+The following picture shows how the Authentication Server is used for AuthZ for the following modules:
+
+- OBaaS Admin (OBaaS CLI server module)
+- Config Server (Manages Config Server Entries)
```
Comment (rhondaday) [inline docs-source/spring/content/security/azn-server/_index.md:130]: "Entries" should be "entries"

File: docs-source/mbaas/content/on-premises/ol8/_index.md [hunk 31]
```diff
@@ -12,20 +12,30 @@ Read [On-Premises](../index.md) and ensure that your desktop meets the minimum s
 
 ### Additional Operating System Packages
 
-Install the following operating system packages:
+As the `root` user, install the following operating system packages:
```
Comment (rhondaday) [inline docs-source/mbaas/content/on-premises/ol8/_index.md:15]: Should 'root' be in blue like it is in line 34?

**Review by andytael — APPROVED** [review]

**Pull request merged**